addons/l10n_ar_withholding/models/afip.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, fields
+
+
+class AfipEarningsTableScale(models.Model):
+    _name = 'afip.earnings_table.scale'
```

Comment:
```suggestion
    _name = 'l10n_ar.earnings.scale'
```
or something better with a prefix